.circleci/config.yml

```diff
@@ -31,19 +31,20 @@ jobs:
       ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY: primary-key
       ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY: deterministic-key
       ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT: derivation-salt
+      EDITOR_ENCRYPTION_KEY: a1b2c3d4e5f67890123456789abcdef0123456789abcdef0123456789abcdef0
```

Comment:
Does this need to go in the env.example as well?

Reply:
thanks! I must have removed it in a stash or something as it used to be there 😕